docs/react/week-1/lesson.md

```diff
@@ -185,14 +168,14 @@ We are also using some shorter syntax within the `HelloWorld` component. `<Greet
 
 Notice how the components that we write (`HelloWorld`, `Greeting`, `Mentor`) are written using a `camel case` convention and always start with an uppercase letter? And "regular DOM" components (`div`, `span`) are always lowercase? This the convention to let you know whether you are using a "regular DOM component" or a component that you have written.
 
-> **Exercise E**:
-> In your `pokedex` React app, open the `src/App.js` file
->
-> 1. Create a new function named `Logo`
-> 2. Copy the `<header>` element and its contents and paste it into the `Logo` component
-> 3. Replace the `<header>` element in the `App` component with the new `Logo` component
-> 4. Create a new component function named `BestPokemon` and return a `<p>` element with some text saying which is your favourite Pokemon (e.g. "My favourite Pokemon is Squirtle")
-> 5. _Render_ your new `BestPokemon` component below the `Logo` component within the `App` component
+| Exercise E (estimate: 5 min)                                                                                                                                                       |
+| :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1. In your `pokedex` React app, open the `src/App.js` file.                                                                                                                        |
+| 2. Create a new function named `Logo`.                                                                                                                                             |
+| 3. Copy the `<header>` element and its contents and paste it into the `Logo` component.                                                                                            |
+| 4. Replace the `<header>` element in the `App` component with the new `Logo` component.                                                                                            |
+| 5. Create a new component function named `BestPokemon` and return a `<p>` element with some text saying which is your favourite Pokemon (e.g. "My favourite Pokemon is Squirtle"). |
+| 6. _Render_ your new `BestPokemon` component below the `Logo` component within the `App` component.                                                                                |
 
 #### Arrow Functions for shorter syntax
```

Comment:
Thanks that's a good spot. I think this is a remnant from before we formatted all the code snippets with Prettier. I believe it used to wrap with parens, but for a single element, Prettier will strip the parens.

So I think there's two options here:

1. We make the examples here have more than 1 element so that Prettier doesn't strip the parens
2. We remove the wording about parens, and assume that Prettier will auto-format the student's code (i.e. lack of knowledge of parens is fine because the tool will handle it for them)

I think I'm leaning towards option 1.

Reply:
Yeah, #1 feels like it's going to be simpler, and reinforces to the students that parens may be required (and if they put more parens than they need, that's not the end of anyone's world)